### PR TITLE
Change delete all button to only display if content exists.

### PR DESCRIPTION
**Pull request by christhompsongoogle**
This is to match other emulator page behavior and align with a11y recommendations to not selectively enable buttons

Internal: b/259448811

### CODE REVIEW

**Review by tonyjhuang — APPROVED** [review]
Thanks for the a11y win!

File: src/components/Storage/Canvas/DeleteAllButton/DeleteAllButton.test.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ describe('DeleteAllButton', () => {
   // TODO: investigate emulators + jsdom
   it.skip('deletes all files', async () => {
```
Comment (ABradham) [inline src/components/Storage/Canvas/DeleteAllButton/DeleteAllButton.test.tsx:29]: It seems like this test wont be run due to the `it.skip` test rule here

**Review by ABradham — APPROVED** [review]
LGTM; Just left a comment about a the test that seems to be skipped in `src/components/Storage/Canvas/DeleteAllButton/DeleteAllButton.test.tsx`.

File: src/components/Storage/Canvas/DeleteAllButton/DeleteAllButton.test.tsx (hunk 1 above)
Comment (christhompsongoogle) [reply src/components/Storage/Canvas/DeleteAllButton/DeleteAllButton.test.tsx:29]: I put a TODO to investigate these soon

**Pull request merged**